cricketscircle/jekyll-test
# adds SEO features to jekyll

## WHAT:
adds SEO features to jekyll
## HOW TO TEST:
click on both and see the page source
http://0.0.0.0:4000/jekyll-test/
http://0.0.0.0:4000/jekyll-test/rachel-dratch/
